docs/source/getting_started.rst

```diff
@@ -0,0 +1,296 @@
+   [2021-07-07 16:44:42,402] [Thread-6] [remoulade.worker.WorkerThread] [INFO] Started Actor transform_city / a0087e8b-0841-4df6-899c-bd59a83d5e98
+   [2021-07-07 16:44:42,402] [Thread-6] [remoulade.worker.WorkerThread] [INFO] Finished Actor transform_city / a0087e8b-0841-4df6-899c-bd59a83d5e98 after 0.02ms.
+   [2021-07-07 16:44:42,513] [Thread-4] [remoulade.worker.WorkerThread] [INFO] Started Actor load_city / a201d61d-5c24-40aa-9b1b-bd8226221804
+   [2021-07-07 16:44:42,513] [Thread-4] [remoulade.worker.WorkerThread] [INFO] Finished Actor load_city / a201d61d-5c24-40aa-9b1b-bd8226221804 after 0.30ms.
+
+Next steps
+------
+
+In this part, we will take a more advanced example to demonstrate how to use Results and Redis to gather results from a group and pipe them to a single actor.
+
+First, you will need to install the dependencies for using Remoulade with Redis::
+
+   $ pip install -U 'remoulade[redis]'
+
+You will also need to install Redis_ or use Docker_ to run a Redis image::
+
+   $ docker run -d --name redis -p 6379:6379 redis
+
+.. _Redis: https://redis.io/
+
+In this example, we want to get the weather for multiple cities, and then load them all into a file by using one single function instead of calling multiple times the ``load_city`` function. In order to achieve this, we have to write a new function : ``load_cities``, which takes a list of text to load instead of a single text::
+
+   @remoulade.actor
+   def load_cities(city_lines):
+       file = open("weather_db.txt", "a")
+       for text in text_list:
+           file.write(text)
+       file.close()
+
+Our get_weather file should now look like this :
+
+.. code-block:: python
+   :caption: get_weather.py
+   :emphasize-lines: 4, 5, 28, 30, 31, 33
+
+
+   import requests
+   import remoulade
+   from remoulade.brokers.rabbitmq import RabbitmqBroker
+   from remoulade.results.backends import RedisBackend
+   from remoulade.results import Results
+
+
+   @remoulade.actor
+   def extract_city(city):
+      ...
+
+
+   @remoulade.actor()
+   def transform_city(response):
+       ...
+
+
+   @remoulade.actor
+   def load_city(text):
+      ...
+
+
+   @remoulade.actor
+   def load_cities(city_lines):
+       ...
+
+
+   result_backend = RedisBackend()
+   rabbitmq_broker = RabbitmqBroker(url="/")
+   result_time_limit_ms = 10 * 60 * 1000 # 10 mn
+   rabbitmq_broker.add_middleware(Results(backend=result_backend, store_results=True, result_ttl=result_time_limit_ms))
+   remoulade.set_broker(rabbitmq_broker)
+   remoulade.declare_actors([extract_city, transform_city, load_city, load_cities])
+
+You will need to restart your Remoulade workers once again after adding this new function.
+Note that we set a time limit, and that in order to be able to gather the results from ``transform_city``, we have to specify "store_results=True" in the Results instantiation.
```

Comment:
```suggestion
Two new things are happening here. First we instantiate a |RedisResBackend| that will allow us to store our data in Redis. The we add a |Results| middleware that will store our results in this backend, with a defined time limit.
```
@antoinerabany what the point of the Results middleware with `store_results=False`?